dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -42,4 +42,5 @@ properties:
       or for K_PRIO_PREEMPT(2)
         priority = <2 PRIO_PREEMPT>;
 
-      When this property is missing a default priority of 0 is assumed.
+      When this property is missing a default priority of <0 PRIO_COOP> is
```

Comment:
`K_PRIO_COOP(0)` will set the highest coop priority. Isn't that rather radical?

Reply:
ok. I guess it won't be obvious to select anyway. I would go for -1 so it would be matching system workqueue (to not starve the system in case of IPC data flood).
I guess if this is a real problem somebody will report it at some point.